### ISSUE TITLE
Revisit the web design

### ISSUE THREAD

**Issue by tpetricek** [tpetricek/FsSnip.Website#10]
The current design is rough copy of the original style with a few changes (I mainly did whatever I could to put something quickly together). It would be nice to redesign the web page and make it nicer.

**Comment by kunjee17** [comment]
there are few current popular other than Bootstrap.

Quite jazzy [Materialize](http://materializecss.com/) and light weight [PureCSS](http://purecss.io/). Have a look. 

and let us know what you like. I m not CSS guy but can workout with frameworks here and there.

**Comment by tpetricek** [comment]
I'm not that worried about the CSS framework we're using - but if switching to another one makes the site nicer, than that's a great thing :). I think it just needs a fresh look from someone who has good design skills, because it all looks a bit ugly.

**Comment by kunjee17** [comment]
@tpetricek I totally agree. But as I told I m more on back end side. It's just I work in front end as day job so I know things or two. So, when I feel that design is appealing to me I just change the framework have better theme template. That is little dumb but works for me. But surely will try to find some help if possible for this.

**Comment by Krzysztof-Cieslak** [comment]
I will try to play a bit with it next few days. I've worked long enough with front-end that I should be able to handle it ;)

And :+1: for Materialize (or any other framework using Material UI style)

**Comment by kunjee17** [comment]
@Krzysztof-Cieslak @tpetricek we need to set up gulp or grunt also. I manually minified current JS files but it will be great to have gulp/grunt integrated in FAKE. It will be easy for other people to modify. What you guys say?

**Comment by kunjee17** [comment]
@Krzysztof-Cieslak I have added package.json with build command in it. It is in PR as of now. But have a look if you have time.

**Comment by Krzysztof-Cieslak** [comment]
Will wait for #58 to be merged before I start doing stuff with web design, having whole system (building, minification, bundling) ready will make things easier.
